From 3fe1eb17a2e6e4bb14cbef0602b9ec85073fe84c Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 18 Sep 2017 16:29:53 +0800
Subject: [PATCH 4518/5242] MLK-19413-7 arm64: dts: fsl-imx8qxp-mek: Enable
 pixel combiner

commit  89a7b01ad842ce80449424334707a20544f696e8 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables pixel combiner for the i.MX8qxp MEK platform.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 5c78ffb..f93d90b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -889,6 +889,10 @@
 	status = "okay";
 };
 
+&pixel_combiner {
+	status = "okay";
+};
+
 &prg1 {
 	status = "okay";
 };
-- 
1.7.9.5

